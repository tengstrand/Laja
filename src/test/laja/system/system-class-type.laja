
#write "target/test-output/system-class-type.out"
  Type of Long: {type("Long").name}
#end
