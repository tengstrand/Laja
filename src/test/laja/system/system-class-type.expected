  Type of Long: java.lang.Long
